MatrixSDK/Crypto/MXOlmDevice.h

```diff
@@ -178,35 +180,32 @@ Determine if an incoming messages is a prekey message matching an existing sessi
 /**
  Generate a new outbound group session.
 
- @return the session id for the outbound session.
+ @param roomId Identifer of the room.
+ @return the newly created outbound group session.
  */
-- (NSString*)createOutboundGroupSession;
+- (OLMOutboundGroupSession *)createOutboundGroupSessionForRoomWithId:(NSString *)roomId;
```

Comment:
```suggestion
- (OLMOutboundGroupSession *)createOutboundGroupSessionForRoomWithRoomId:(NSString *)roomId;
```

For convention naming and better integration with swift.